File: benchmark/equiv/TrailingZeroes3.java
public class TrailingZeroes3 {
  static public int trailingZeroes1(int n) {
    int num;
    for (num = 0; n != 0; n /= 5, num += n)
      ;
    return num;
  }

  static public int trailingZeroes3(int n) {
    int s = 0;
    while (n > 4)
      s += (n /= 5);
    return s;
  }
}

File: src/equiv/TrailingZeroes3.ml
module JB = Javalib_pack.JBasics
module QID = QualifiedIdentity
module E = Expr

let _ =
  let x = E.Var (Var.Mk (QID.QID ["x"], Type.Int)) in

  Equiv.run "TrailingZeroes3"
            [(JB.TBasic `Int)]
            (JB.TBasic `Int)
            "trailingZeroes1"
            "trailingZeroes3"
            [x]
            [E.mk_ige x (E.Int 5)]
